feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/novacard/overview/webViewController/NovaCardWebPageProvider.kt

```diff
@@ -0,0 +1,54 @@
+package io.novafoundation.nova.feature_assets.presentation.novacard.overview.webViewController
+
+class NovaCardWebPageProvider(
+    private val widgetId: String,
+    private val refundAddress: String
+) {
+
+    private val containerId = "widget-container"
+    private val callbackName: String = "NovaCallback"
+
+    fun getCallbackName() = callbackName
+
+    fun getPage(): String {
+        return """
+        <!DOCTYPE html>
+        <html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        </head>
+        <body>
+            <div id="$containerId"></div>
+        
+            <script src="https://widget.mercuryo.io/embed.2.0.js"></script>
+        </body>
+        </html>
+        """.trimIndent()
+    }
+
+    fun getJsScript(): String {
+        return """
+            mercuryoWidget.run({ 
+                widgetId: '$widgetId',
+                host: document.getElementById('widget-container'),
+                type: 'sell',
+                currency: 'DOT',
```

Comment:
I think hardcoded DOT token is okay for now but as an another way we can provide it here from view model. What do you think?